backend/migrate_calibration.py

```
@@ -0,0 +1,135 @@
+Calibration Database Migration Script for SupplyLine MRO Suite
+
+This script creates the calibration-related database tables that were added in v3.5.0.
+Run this script to add calibration functionality to existing installations.
+
+Usage:
+    python migrate_calibration.py [environment]
+
+Where environment is one of: development, production (default: development)
+"""
+
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)